foxtrot-core/src/main/java/com/flipkart/foxtrot/core/table/TableManager.java

```diff
@@ -0,0 +1,22 @@
+package com.flipkart.foxtrot.core.table;
```

Comment:
License .. this is not required now, but before release we need to ensure all source files have license.